frontend/js/firstPage.js

```diff
@@ -61,8 +61,13 @@ function authorEntered (e) {
     datetime: Date.now(), // number, unicode timestamp
     location: [crd.latitude, crd.longitude], // Array of GPS coordinates
     text: poem, // string
+<<<<<<< HEAD
+    author, // string,
+    route: chosenRoute // number, a bus route
+=======
     author: author, // string,
     route: Number(chosenRoute) // number, a bus route
+>>>>>>> e6bc869222e3eb65379520b2ce0769ce959bf945
```

Comment:
these are git makers, they show up in code when there's a merge conflict, u'll need to clean this up before the home page will work. replace this function with the following:

```js
function authorEntered (e) {
  const textField = document.querySelector('#textid')
  author = textField.value
  addBubble(author, 'bubble2')
  textField.value = ''
  postPoem({
    datetime: Date.now(), // number, unicode timestamp
    location: [crd.latitude, crd.longitude], // Array of GPS coordinates
    text: poem, // string
    author: author, // string,
    route: Number(chosenRoute) // number, a bus route
  })
}
```

that's the way the function looks on the class website